skimage/__init__.py

```diff
@@ -56,18 +56,21 @@
 
 """
 
-import os.path as _osp
-import imp as _imp
-import functools as _functools
-import warnings as _warnings
+import os.path as osp
+import imp
+import functools
+import warnings
```

Comment:
I think these were aliased previously to hide them from the base package namespace.